subsys/bluetooth/host/gatt.c

```diff
@@ -4860,7 +4860,8 @@ void bt_gatt_connected(struct bt_conn *conn)
 	 * does not have an LTK indicating that the client has lost the bond,
 	 * enabling encryption will fail.
 	 */
-	if (IS_ENABLED(CONFIG_BT_SMP) &&
+	if (IS_ENABLED(CONFIG_BT_SMP) && (conn->role == BT_HCI_ROLE_MASTER ||
+	    IS_ENABLED(CONFIG_BT_GATT_AUTO_SEC_REQ)) &&
```

Comment:
I'm not sure this is anywhere explicitly stated, but style-wise I tend to align split lines with the innermost opening brace that the split line belongs to, i.e. in this case with `conn->role` on the first line. At least I find it easier to see how the grouping of the conditions goes.

Reply:
Agreed. 

Basically
```
if (bool_expr_1 && (bool_expr_2 &&
		    bool_expr_3)) {
}
```
or perhaps even better (if applicable): 
```
if (bool_expr_1 && 
    (bool_expr_2 && bool_expr_3)) {
}
```
else 
```
if (bool_expr_1 &&
    (bool_expr_2 &&
     bool_expr_3)) {
}
```